src/utils/weatherService.ts

```diff
@@ -0,0 +1,31 @@
+export interface Weather {
```

Comment:
Feels like this is a network request to your an api so I probably wouldn't call it a util and since it only making a request, I would probably also not call it a service, maybe instead: `src/api/weather.ts`.